charts/posthog/values.yaml

```diff
@@ -242,6 +241,27 @@ email:
   # existingSecret: secret-name
   # existingSecretKey: smtp-password
 
+# SAML
+saml:
+  # -- Whether password-based login is disabled and users automatically redirected to SAML login. Requires SAML to be properly configured.
+  enforced: false
+  # -- Whether SAML should be completely disabled. If set at build time, this will also prevent SAML dependencies from being installed.
+  disabled: false
```

Comment:
shouldn't this be `true` by default?

Reply:
also the convention seems to be to rather have `saml.enabled: true/false`

Reply:
Actually no, we only support disabling SAML in case you have an environment that doesn't support the needed deps (like Heroku) or don't want to install those deps for some reason, but it should be a rare case in this context.

As for `saml.disabled` it's following the convention of the env var that we read in the codebase (see `SAML_DISABLED` in https://posthog.com/docs/self-host/configure/environment-variables), but happy to change if you think it's clearer.

Reply:
This seems reasonable enough to me with the comments